app/questionnaire/rules.py

```diff
@@ -302,13 +323,19 @@ def evaluate_when_rules(when_rules, schema, metadata, answer_store, group_instan
     return True
 
 
-def get_answer_store_value(answer_id, answer_store, schema, group_instance, group_instance_id=None):
+def get_answer_store_value(answer_id, answer_store, schema, group_instance, group_instance_id=None, routing_path=None):
+
     filtered = answer_store.filter(answer_ids=[answer_id])
 
-    if not filtered.count():
+    if routing_path:
```

Comment:
Do we ever want to not use the routing path?

Reply:
Idea here was to avoid refactoring everything in runner to get the routing path